utils.zsh/30-build.zsh

```diff
@@ -14,10 +14,12 @@ setup_target() {
     [output_dir]="${project_root}/${1}/obs-${PACKAGE_NAME}-${1##*-}"
   )
 
+  mkdir -p "${project_root}/${1}/obs-${PACKAGE_NAME}-${1##*-}"
+
   case ${1} {
     macos-x86_64)
       config_data+=(
-        [deployment_target]=10.13
+        [deployment_target]=10.15
```

Comment:
Is this change intentional/necessary?

Reply:
Yes, @WizardCM and I talked about raising this for 10.15 anyway with macOS 13 releasing this year.

Also Qt6 complains about a 10.13 target because it requires 10.14 at the very least.

Reply:
As discussed off-thread we will raise the minimum supported version to 10.15 with the next OBS release, so this aligns.